packages/plugin-infra/src/hooks/RemoteView.ts

```diff
@@ -0,0 +1,23 @@
+import React, { createElement, Fragment, useMemo } from 'react'
+import { useActivatedPluginsSNSAdaptor } from '../manager/sns-adaptor'
+import type { Plugin, PluginId } from '../types'
+
+export interface RemoteViewProps<Name extends keyof Plugin.SNSAdaptor.ContributedViewRegistry> {
+    name: Name
+    pluginID?: PluginId
+    fallback?: React.ReactElement | null
+}
+export const RemoteView: <Name extends keyof Plugin.SNSAdaptor.ContributedViewRegistry>(
```

Comment:
We could regulate the `RemoteView` as plugin widgets. Such a concept provides a standalone feature that is more reasonable for reusing across plugins and less likely to abuse usage than `RemoteView`.

Reply:
Can you explain? I didn't get that